chore(github): add some tips to improve issues and PRs (#4268)
spinnaker/spinnaker: .github/ISSUE_TEMPLATE.md

* Description: The behavior you expect to see, and the actual behavior.
* Steps to reproduce: Ideally, an isolated way to reproduce the behavior (example: GitHub repository with code isolated to the issue that anyone can clone to observe the problem). If not possible, as much information as possible to see this behavior.
* Additional Details: Additional information such as screenshots and exception logs.

---
# Helpful Tips

## Add screenshots and annotations
<kbd>![](https://cl.ly/744faf8f96d7/download/Image%202019-04-08%20at%2012.35.32.png)</kbd>
- In OSX Mojave, you can capture screenshots and add annotations. https://support.apple.com/en-us/HT201361
- Another app you can use is https://www.getcloudapp.com/.

> ## Add a gif to show an action

Reviewer: ```suggestion
## Add an animated gif to show an action
```

> GIFs are great to show an action, however they sometimes move too fast. 

Reviewer: ```suggestion
Animated gifs are a great way to show action; however, they sometimes move too fast.
```
(getting around inconsistent capitalization of `gif`)

One way to improve them is to capture 3 pieces of content:
- screenshot before an action
- screenshot after the action

> - gif of the action

Reviewer: ```suggestion
- animated gif of the action
```

> - You can edit a gif using https://ezgif.com/

Reviewer: Should this be "create" instead of "edit"?

Author: create/edit?

I've been recording a gif on my computer, then editing it on ezgif.com

Reviewer 2: in this context I'd agree with @anotherchrisberry that create is right

## Put a border around a screenshot/gif
Add a border to help improve the readability.
```html
<kbd>![](https://www.spinnaker.io/assets/images/spinnaker-logo-inline.svg)</kbd>
```

## Add expandable sections
<details>

> <summary>Add expandable sections to hide away large code blocks/images or reduce the motion of a gifs</summary>

Reviewer: ```suggestion
<summary>Add expandable sections to hide away large code blocks/images or reduce the motion of a gif</summary>
```
